### DISCUSSION TITLE
Compatiblity with ESP-ECU

### DISCUSSION THREAD

**Discussion by JochenKlenk** [ksheumaker/homeassistant-apsystems_ecur#120] [Ideas]
Hi guys,

would be amazing if we can get this compatible with ESP-ECU which emulate the ECU on an ESP with an CC2530 connected.

https://github.com/patience4711/read-APSystems-YC600-QS1-DS3

What you think about that?

Thanks for the help in advance.

**Comment by doug-last** [comment]
on an similar subject, maybe someday we will be able to get the inverter data from the zigbee usb dongle plugged directly in homeassistant without the ecu (or esp in this case)
homeassistant already uses zigbee to get data from smart-things, using "bellows dump" from zigpy in HA i can see the packets between inverters and ecu, i'm trying to learn how all this works, but i believe it will be easier when the inverter get added to "zigbee-herdsman" project, that is used by zigbee2mqtt that is used on HA, or an 'quirk' for zigpy.
i might be completely wrong tho.
